src/common/legacy/src/cnn_network_impl.cpp

```diff
@@ -349,7 +349,7 @@ void CNNNetworkImpl::removeOutput(const string& dataName) {
     }
 }
 
-size_t CNNNetworkImpl::getBatchSize() const noexcept {
+size_t CNNNetworkImpl::getBatchSize() const {
```

Comment:
Why did you remove noexcept for all methods?
at() throws an exception? Or something else?

Reply:
There was a false positive from algorithm perspective - coverity shows the case which causes exception, but it was handled above in a code. Reverted